README.md

```diff
@@ -1,24 +1,20 @@
-# Inter Protocol UI
+# dapp-psm
 
-User application for Agoric Inter Protocol--Vaults, BLD Boost, etc.
+UI for Inter Protocol Vaults, BLD Boost, Liquidations, etc.
```

Comment:
wby lose the dash? it expands the functionality of Inter Protocol

Reply:
Wasn't intentional, fixed. I just copied the readme from dapp-psm and forgot what was here originally